js/ui/popup.js

```diff
@@ -666,29 +672,54 @@ var Popup = Overlay.inherit({
                 + popupHeightParts.popupVerticalOffsets,
             overlayContent = this.overlayContent().get(0),
             cssStyles = {},
+            isAutoWidth = this.overlayContent().get(0).style.width === "auto" || this.overlayContent().get(0).style.width === "",
             contentMaxHeight = this._getOptionValue("maxHeight", overlayContent),
-            contentMinHeight = this._getOptionValue("minHeight", overlayContent),
-            isAutoResizable = !isIE11 || (!contentMaxHeight && !contentMinHeight);
-
-        if(this.option("autoResizeEnabled") && this._isAutoHeight() && isAutoResizable) {
-            if(!isIE11) {
-                var container = $(this._getContainer()).get(0),
-                    maxHeightValue = sizeUtils.addOffsetToMaxHeight(contentMaxHeight, -toolbarsAndVerticalOffsetsHeight, container),
-                    minHeightValue = sizeUtils.addOffsetToMinHeight(contentMinHeight, -toolbarsAndVerticalOffsetsHeight, container);
-
-                cssStyles = extend(cssStyles, {
-                    minHeight: minHeightValue,
-                    maxHeight: maxHeightValue
-                });
+            contentMinHeight = this._getOptionValue("minHeight", overlayContent);
+
+        var currentHeightStrategyClass = heightStrategies.static;
+
+        if(this._isAutoHeight() && this.option("autoResizeEnabled")) {
+            if(isAutoWidth || isOldSafari) {
+                if(isIE11) {
+                    currentHeightStrategyClass = heightStrategies.static;
```

Comment:
You've already set this value to a variable at [R679](https://github.com/DevExpress/DevExtreme/pull/8340/files#diff-353a433c5241a480804acc2b08b8330fR679)